articles/architecture-scenarios/_includes/_keep-reading.md

```diff
@@ -0,0 +1,78 @@
+<% if (platform === "b2b") { %>
+<!-- Exclude current page from keep reading section -->
+<% if (self !== "architecture") { %>
+* [Architecture](/architecture-scenarios/implementation/b2b/b2b-architecture)
+<% } %>
+<% if (self !== "provisioning") { %>
+* [Provisioning](/architecture-scenarios/implementation/b2b/b2b-provisioning)
+<% } %>
+<% if (self !== "authentication") { %>
+* [Authentication](/architecture-scenarios/implementation/b2b/b2b-authentication)
+<% } %>
+<% if (self !== "branding") { %>
+* [Branding](/architecture-scenarios/implementation/b2b/b2b-branding)
+<% } %>
+<% if (self !== "deployment") { %>
+* [Deployment Automation](/architecture-scenarios/implementation/b2b/b2b-deployment)
+<% } %>
+<% if (self !== "qa") { %>
+* [Quality Assurance](/architecture-scenarios/implementation/b2b/b2b-qa)
+<% } %>
+<% if (self !== "profile-mgmt") { %>
+* [Profile Management](/architecture-scenarios/implementation/b2b/b2b-profile-mgmt)
+<% } %>
+<% if (self !== "authorization") { %>
+* [Authorization](/architecture-scenarios/implementation/b2b/b2b-authorization)
+<% } %>
+<% if (self !== "operations") { %>
+* [Operations](/architecture-scenarios/implementation/b2b/b2b-operations)
+<% } %>
+<% if (self !== "logout") { %>
+* [Logout](/architecture-scenarios/implementation/b2b/b2b-logout)
+<% } %>
+<% if (self !== "operations") { %>
+* [Operations](/architecture-scenarios/implementation/b2b/b2b-operations)
+<% } %>
+<% if (self !== "launch") { %>
+* [Launch Preparation](/architecture-scenarios/implementation/b2b/b2b-launch)
+<% } %>
+<%  } %>
+
+<% if (platform === "b2c") { %>
```

Comment:
Once done making the above change, you can kill this whole block...

Reply:
As I explained in first comment, interpolation and markdown links doesn't play well with our docs "Compiler". That's why I chosen this fallback option.